docs/sources/datasources/aws-cloudwatch/query-editor/index.md

```diff
@@ -218,9 +218,17 @@ When making `stats` queries in [Explore]({{< relref "../../../explore/" >}}), ma
 
 {{< figure src="/static/img/docs/v70/explore-mode-switcher.png" max-width="500px" class="docs-image--right" caption="Explore mode switcher" >}}
 
+## Cross-account observability
+
+The CloudWatch plugin provides the ability to monitor and troubleshoot applications that span multiple accounts within a region. Using cross-account observability, you can seamlessly search, visualize and analyze metrics and logs without worrying about account boundaries.
+
 ### Getting started
 
-To enable cross-account observability, first enable it in CloudWatch using the official [CloudWatch docs](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Unified-Cross-Account.html), then add [two new API actions]({{< relref "../#cross-account-observability" >}}) to the IAM policy attached to the role/user running the plugin.
+To enable cross-account observability do the following:
+
+1. Go to the [Amazon CloudWatch docs](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Unified-Cross-Account.html) and follow the instructions on enabling cross-account observability.
+
+1. Add [two new API actions](/docs/grafana/latest/datasources/aws-cloudwatch/#cross-account-observability-permissions) to the IAM policy attached to the role/user running the plugin.
```

Comment:
question about this one. Is the name of the section you are linking to called "two new API actions"? We want the link name to be the heading title. Also the concept of 'adding new' makes my mind melt because the word 'add' implies 'new', so 'add new' is redundant. Can this just be 'add two api actions'?